### PR TITLE
Home.java:47-48: Prohibit absolute paths in methods

### PR DESCRIPTION
**Pull request by cosmincalinescu91**

### CODE REVIEW

File: eo-maven-plugin/src/main/java/org/eolang/maven/Home.java [hunk 1]
```diff
@@ -197,6 +199,10 @@ public static Path clean(final Path path) {
      * @return Absolute path
      */
     private Path absolute(final Path path) {
+        if (path.isAbsolute()) {
+            throw new IllegalArgumentException(format("Absolute path %s is prohibit.", path));
```
Comment (volodya-lombrozo) [inline eo-maven-plugin/src/main/java/org/eolang/maven/Home.java:203]: I'm not sure that we have to throw an exception in that case. If the `path` parameter is an absolute path then this method trivially returns `path` as is. It's a behaviour of the `Path.resolve();` method.

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
Hi @cosmincalinescu91. The reason isn't so clear, why we have to throw an exception in that case. Maybe it's better to start from the issue and provide some context for that change? Also, it's Important to make all tests and checks pass before submitting PR.
In order to do so, please, run the next command:
```
 mvn clean install -Pqulice
```
For more information please check [how to contribute](https://github.com/objectionary/eo#how-to-contribute).
Thank you for your contribution.

**Comment by mximp** [comment]
@cosmincalinescu91 please always add issue id (`#XXX`) into PRs description - it will link the PR and the issue.
Also when PR is ready to be reviewed please tag a user you want to review the PR.

File: eo-maven-plugin/src/main/java/org/eolang/maven/Home.java (hunk 1 above)
Comment (mximp) [inline eo-maven-plugin/src/main/java/org/eolang/maven/Home.java:203]: @cosmincalinescu91 I believe more meaningful would be: "Absolute paths are prohibited: %s"

**Review by mximp — CHANGES_REQUESTED** [review]

File: eo-maven-plugin/src/main/java/org/eolang/maven/Home.java (hunk 1 above)
Comment (mximp) [reply eo-maven-plugin/src/main/java/org/eolang/maven/Home.java:203]: @volodya-lombrozo exactly. And if you check `this.absolute` is used in all three methods.

**Comment by yegor256** [comment]
@cosmincalinescu91 what's up with this? will you continue?

**Comment by yegor256** [comment]
@cosmincalinescu91 planning to continue or we can close this PR?

**Comment by yegor256** [comment]
@volodya-lombrozo what's up with this one? We should continue or maybe just close?

**Comment by volodya-lombrozo** [comment]
@cosmincalinescu91, since we haven't received any response from you, we regretfully had to close that PR. We would greatly appreciate it if you could take into consideration the comments provided in the current PR for any future ideas or suggestions. Your understanding in this matter would mean a lot to us, and we eagerly await your valuable suggestions and PR. Thank you for your attention and cooperation

**Pull request closed without merging**